src/test/resources/052.xsd

```diff
@@ -0,0 +1,12 @@
+<?xml version = "1.0"?>
+<xs:schema xmlns:xs = "http://www.w3.org/2001/XMLSchema">
+    <xs:element name="root">
+        <xs:complexType> 
+            <xs:sequence>
+                <xs:element type="xs:integer" name="ProductID"/>
+                <xs:element type="xs:string" name="ProductName"/>
+                <xs:element type="xs:decimal" name="Price"/>
```

Comment:
I think this is losing information.

We started with a Connect schema that specified a scale of 3 (three fraction digits). [ref](https://github.com/ibm-messaging/kafka-connect-xml-converter/pull/1/files#diff-131544f6bb7168f4bb1ce187a736e6448b89929ac2c6519655657d7526a5dab0R1691)

But the schema here doesn't mention that. 

We should try to ensure that the Connect schema is fully reflected in the XML schema. 

  ```suggestion
                  <xs:element name="Price">
                    <xs:simpleType>
                        <xs:restriction base="xs:decimal">
                            <xs:fractionDigits value="3"/>
                        </xs:restriction>
                    </xs:simpleType>
                </xs:element>
```

ref: https://learning.oreilly.com/library/view/xml-schema/0596002521/re24.html

Reply:
updated